src/screenControl.py

```diff
@@ -473,8 +480,8 @@ def resetDirty(self):
         self.linesDirty = False
         self.dirtyIndexes = []
 
-    def dirtyHoverIndex(self):
-        self.dirtyIndexes.append(self.hoverIndex)
+    def dirtyHoverIndex(self, idx=-1):
+        self.dirtyIndexes.append(idx if idx > 0 else self.hoverIndex)
```

Comment:
probably should make this a separate function rather than have the optional negative-default peram